scipy/optimize/_zeros_py.py

```diff
@@ -1393,3 +1393,189 @@ def toms748(f, a, b, args=(), k=1,
     x, function_calls, iterations, flag = result
     return _results_select(full_output, (x, function_calls, iterations, flag),
                            "toms748")
+
+
+def _muller(f, x0, x1, x2, args=(), tol=1.48e-8, maxiter=50,
+           rtol=0.0, full_output=False, disp=True):
+    """
+    Find a root of a real or complex function using the Muller's method.
+
+    Find a root of the scalar-valued function `func` given a nearby scalar
+    starting points `x0`, `x1`, and `x2`.
+    The Muller method requires three starting points, but no derivatives,
+    and it works well for complex-valued roots. It will converge towards
+    complex roots even with real starting points. As with other root-finding
+    methods, initial guesses should be distinct.
+
+    `muller` is for finding roots of a scalar-valued functions of a single
+    variable. For problems involving several variables, see `root`.
+
+    Parameters
+    ----------
+    f : callable
+        The function whose root is wanted. It must be a function of a
+        single variable of the form ``f(x,a,b,c...)``, where ``a,b,c...``
+        are extra arguments that can be passed in the `args` parameter.
+        The function can be either real- or complex-valued, but it should 
+        support complex numbers, as intermediate guesses will be complex.
```

Comment:
```suggestion
        support complex input, as intermediate guesses will be complex.
```
Is this that is meant?